modules/services/agentless-scanning/README.md

```diff
@@ -6,7 +6,6 @@ The following resources will be created in each instrumented account:
 - An IAM Role and associated policies that allows Sysdig to perform tasks necessary for agentless scanning.
 - A KMS key used to transcript volume snapshots in the each region. Alias for this key in each region.
 
-<!-- BEGINNING OF PRE-COMMIT-TERRAFORM DOCS HOOK -->
```

Comment:
Is the removal of these lines intentional? I believe these are required by pre-commit

Reply:
As far as I know the pre-commit hook has been deprecated, that's the reason why I removed these lines and updated the contribution guidelines in order to explain how the documentation should be generated. If that's not the case, I can simply remove the single commit where the related changes are now contained

Reply:
> the pre-commit hook has been deprecated

I don't think that's true. There were some broken pre-commit hooks previously and I over-zealously removed all of them. The auto-updates via terraform-docs are valuable, should be re-added, although that does not have been done in this PR.